seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/TextWriteStrategy.java

```diff
@@ -133,15 +133,21 @@ private FSDataOutputStream getOrCreateOutputStream(@NonNull String filePath) {
                         OutputStream out =
                                 lzo.createOutputStream(fileSystemUtils.getOutputStream(filePath));
                         fsDataOutputStream = new FSDataOutputStream(out, null);
+                        fsDataOutputStream.write(String.join(",", seaTunnelRowType.getFieldNames()).getBytes());
+                        fsDataOutputStream.write(System.lineSeparator().getBytes());
```

Comment:
Use `fsDataOutputStream.write(rowDelimiter.getBytes());` ?

Reply:
yeah,i will fix it